### PR TITLE
Add flag for stations that need a country hint

### PR DESCRIPTION
**Pull request by inouire**
Keep comments as is for the moment

### CODE REVIEW

**Comment by espadrine** [comment]
Could you paste the script that generated the change here, and re-run it to rebase your work?

**Comment by inouire** [comment]
Rebased. My script is part of a bigger framework about stations manipulation, but here is the main part for this case:

```ruby
COUNTRY_TRANSLATIONS  = {
  :de => [ "spanien", "frankreich", "england", "irland", "schottland", "italien", "belgien", "deutschland", "portugal", "ungarn", "niederlande", "weißrussland", "schweiz", "tschechien", "dänemark", "slowenien", "schweden" ],
  :en => [ "spain", "france", "england", "ireland", "scotland", "italy", "belgium", "germany", "portugal", "hungary", "netherlands", "belarus", "switzerland", "czech republic", "denmark", "slovenia", "sweden" ],
  :es => [ "españa", "francia", "inglaterra", "irlanda", "escocia", "italia", "bélgica", "alemania", "portugal", "hungría", "países bajos", "bielorrusia", "suiza", "república checa", "dinamarca", "eslovenia", "suecia" ],
  :fr => [ "espagne", "france", "angleterre", "irlande", "écosse", "italie", "belgique", "allemagne", "portugal", "hongrie", "pays-bas", "biélorussie", "suisse", "république tchèque", "danemark", "slovénie", "suède" ],
  :it => [ "spagna", "francia", "inghilterra", "irlanda", "scozia", "italia", "belgio", "germania", "portogallo", "ungheria", "paesi bassi", "bielorussia", "svizzera", "repubblica ceca", "danimarca", "slovenia", "svezia"],
}

def is_hinted?(station)
  COUNTRY_TRANSLATIONS.any? do |key, names|
    names.include?((station.comments[key] || '').strip.downcase)
  end
end

hinted_stations = stations.flat_map do |station|
  if is_hinted?(station)
    ([station] + station.get_children).map do |to_hint|
      puts "[x] ".cyan + station.to_s
      to_hint
    end
  end
end.uniq.compact
```

File: test_data.rb [hunk 1]
```diff
@@ -469,10 +469,22 @@ def test_airports_comments_and_children
...
+        assert child_row['is_airport'] == 't', "#{child_row['name']} (#{child_row['id']}) should be an airport, as a child of airport station #{row['name']} (#{row['id']})"
+      end
+    end
+  end
+
+  def test_country_hints_for_children
+    STATIONS.select do |row|
+      row['country_hint'] == 't'
+    end.each do |row|
+      # TODO once info are cleaned: check that the comment does not contain country information
```
Comment (espadrine) [inline test_data.rb]: IIUC, `info:` comments that only include the country will be removed in a follow-up, warranting a new check that `info:` comments not just be a country name.

Do I understand correctly?

Comment (inouire) [inline test_data.rb]: Absolutely. We cannot get rid of the info: comments about countries while the API does not rebuild it on the fly thanks to the new flag. Same goes for `is_airport` btw

**Comment by inouire** [comment]
Thanks !

**Pull request merged**